ArduCopter/mode.cpp

```diff
@@ -627,15 +629,15 @@ Copter::Mode::AltHoldModeState Copter::Mode::get_alt_hold_state(float target_cli
     // Alt Hold State Machine Determination
     if (!motors->armed()) {
         // the aircraft should moved to a shut down state
-        motors->set_desired_spool_state(AP_Motors::DESIRED_SHUT_DOWN);
+        motors->set_desired_spool_state(AP_Motors::DesiredSpoolState::SHUT_DOWN);
 
         // transition through states as aircraft spools down
-        switch (motors->get_spool_mode()) {
+        switch (motors->get_spool_state()) {
 
-        case AP_Motors::SHUT_DOWN:
+        case AP_Motors::SpoolState::SHUT_DOWN:
             return AltHold_MotorStopped;
 
-        case AP_Motors::DESIRED_GROUND_IDLE:
+        case AP_Motors::SpoolState::GROUND_IDLE:
```

Comment:
So this was using the wrong enum but luckily "DESIRED_GROUND_IDLE" and "GROUND_IDLE" were both enumerated as '1".

Reply:
Yes.